server/src/test/java/io/crate/integrationtests/InsertIntoIntegrationTest.java

```diff
@@ -2018,4 +2018,33 @@ sub int as round((random() + 1) * 100)
             assertThat(response.rows()[0][0]).isEqualTo(generated);
         }
     }
+
```

Comment:
Can we add a comment that this tests a regression and link to the related issue?

Reply:
I removed Itest and added a `TransportShardUpsertActionTest`. Added issue link to the new test